api/rolestore/model.go

```diff
@@ -7,339 +7,483 @@
+	Comment                string          `json:"comment,omitempty"`
+	Tags                   []string        `json:"tags,omitempty"`
+	Roles                  []Role          `json:"roles"`
+	Attributes             []Attributes    `json:"attributes"`
+	Permissions            []string        `json:"permissions"`
+	FirstName              string          `json:"first_name,omitempty"`
+	LastName               string          `json:"last_name,omitempty"`
+	FullName               string          `json:"full_name,omitempty"`
+	JobTitle               string          `json:"job_title,omitempty"`
+	Company                string          `json:"company,omitempty"`
+	Department             string          `json:"department,omitempty"`
+	Email                  string          `json:"email,omitempty"`
+	Telephone              string          `json:"telephone,omitempty"`
+	DistinguishedName      string          `json:"distinguished_name,omitempty"`
+	Locale                 string          `json:"locale,omitempty"`
+	SamAccountName         string          `json:"samaccountname,omitempty"`
+	WindowsAccount         string          `json:"windows_account,omitempty"`
+	UnixAccount            string          `json:"unix_account,omitempty"`
+	Password               string          `json:"password,omitempty"`
+	MFA                    MFAStatus       `json:"mfa,omitempty"`
+	Settings               json.RawMessage `json:"settings,omitempty"`
+	ExternalID             string          `json:"external_id,omitempty"`
+	AuthorizedKeys         []AuthorizedKey `json:"authorized_keys,omitempty"`
+	WebAuthnCredentials    []Credential    `json:"webauthn_credentials,omitempty"`
+	SessionPasswordEnabled bool            `json:"session_password_enabled,omitempty"`
+	StaleAccessToken       bool            `json:"stale_access_token,omitempty"`
+	CurrentSessionID       string          `json:"current_session_id,omitempty"`
+	RefreshTimestamp       string
+}
+
+// Credential webauthn credential definition.
+type Credential struct {
+	ID           string `json:"id"`
+	CredentialID string `json:"credential_id"`
+	Name         string `json:"name,omitempty" diff:"name"`
+	Comment      string `json:"comment,omitempty" diff:"comment"`
+	LastUsed     string `json:"last_used,omitempty"`
+	Created      string `json:"created,omitempty"`
+	Author       string `json:"author,omitempty"`
+	Updated      string `json:"updated,omitempty"`
+	UpdatedBy    string `json:"updated_by,omitempty"`
+}
+
+// MFAStatus mfa status definition.
+type MFAStatus struct {
+	Status          string  `json:"user_mfa_status,omitempty"`
+	TotpStatus      string  `json:"status,omitempty"`
+	TotpMFASeed     MFASeed `json:"seed,omitempty"`
+	MobileMfaStatus string  `json:"mobile_mfa_status"`
+}
+
+type MFASeed struct {
+	Seed_string  string `json:"seed_string,omitempty"`
+	Seed_qr_code string `json:"seed_qr_code,omitempty"`
+}
+
+// Attributes user attributes definition.
+type Attributes struct {
```

Comment:
UserAttribute (also, not plural)

Reply:
That has been taken from the original struct definition, but let's name it differently in the sdk, UserAttribute makes more sense in this case.